README.adoc

```diff
@@ -140,21 +140,23 @@ docker build -t gateway:1.0-SNAPSHOT gateway/.
 
 Next, use the provided script to start the application in Docker containers. The script creates a network for the containers to communicate with each other. It also creates containers for Kafka, Zookeeper, and all of the microservices in the project.
 
-****
-[system]#*{linux} | {mac}*#
+include::{common-includes}/os-tabs.adoc[]
 
+[.tab_content.mac_section.linux_section]
+--
 [role='command']
 ```
 ./scripts/start-app
 ```
+--
 
-[system]#*{win}*#
-
+[.tab_content.windows_section]
+--
 [role='command']
 ```
 .\scripts\start-app.bat
 ```
-****
+--
```

Comment:
Windows tab displays nothing:
![image](https://user-images.githubusercontent.com/26514093/66077956-0f094480-e52f-11e9-8a0e-1256e092954c.png)

Reply:
Works for me 
<img width="574" alt="image" src="https://user-images.githubusercontent.com/25497083/66078082-52fc4980-e52f-11e9-9a5c-856669abd3de.png">

Reply:
Just worked for me!